components/network/freeradius/Makefile

```diff
@@ -50,22 +50,26 @@ TEST_TARGET= $(NO_TESTS)
 
 COMPONENT_PRE_CONFIGURE_ACTION += ( \
         $(CLONEY) $(SOURCE_DIR) $(@D); \
+	$(GSED) -i -e 's/-D_LIBRADIUS/-D_LIBRADIUS -D_XPG4_2/' $(@D)/src/lib/all.mk; \
         );
 
-CONFIGURE_OPTIONS += CFLAGS="-D_XPG4_2 -I/usr/include/openldap -I/usr/include/odbc -I/usr/postgres/$(PG_VERSION)/include"
+CONFIGURE_OPTIONS += CFLAGS="-I/usr/include/odbc"
 CONFIGURE_OPTIONS += LDFLAGS="-lldap_r"
 CONFIGURE_OPTIONS += --srcdir=$(BUILD_DIR)/$(MACH64)
 CONFIGURE_OPTIONS += --sysconfdir=$(ETCDIR)
 CONFIGURE_OPTIONS += --localstatedir=$(VARDIR)
 CONFIGURE_OPTIONS += --with-pcap-lib-dir=$(USRLIBDIR)
 CONFIGURE_OPTIONS += --with-pcap-include-dir=/usr/include/pcap
+CONFIGURE_OPTIONS += --with-rlm-ldap-include-dir=/usr/include/openldap
+CONFIGURE_OPTIONS += --with-rlm-sql_postgresql-include-dir=/usr/postgres/$(PG_VERSION)/include
 CONFIGURE_OPTIONS += --with-openssl-lib-dir=$(OPENSSL_LIBDIR)
```

Comment:
You are using files for the build that are part of $(PG_DEVELOPER_PKG). Thus, you should manually require this package (and also add an appropriate comment).